tests/test_mem_eff_attention.py

```diff
@@ -823,3 +826,45 @@ def test_cuda_streams(
         atol=op.FORWARD_ERROR_ATOL[dtype],
         rtol=op.FORWARD_ERROR_RTOL.get(dtype, 1e-5),
     )
+
+
+@cuda_only
+@pytest.mark.parametrize("seed", [42])
+@pytest.mark.parametrize("k_len", [32])
+@pytest.mark.parametrize("batch_size", [2])
+@pytest.mark.parametrize("kv_len", [32])
+@pytest.mark.parametrize("q_len", [33])
+@pytest.mark.parametrize("device", ["cuda"])
```

Comment:
Can we also parametrize on the operator here? Just to make sure this tests all of the operators that support custom scaling. Currently we have `op=None` so this will only test the default operator for these shapes.

Reply:
Good suggestion. Fixed.